packages/sdk/package.json

```diff
@@ -73,6 +73,7 @@
     "escape-string-regexp": "4.0.0",
     "execa": "5.1.1",
     "find-up": "5.0.0",
+    "fs-jetpack": "4.3.0",
```

Comment:
Now that it's a dependency, it can probably be removed as a devDependency (line 51)

Reply:
That's what I did I think 🙈 but I messed up the merge conflict I guess